notebook/templates/page.html

```diff
@@ -124,6 +124,236 @@
 
 <div id="header" role="navigation" aria-label="{% trans %}Top Menu{% endtrans %}">
   <div id="header-container" class="container">
+  <input id="checkbox-sidebar" class="checkbox-sidebar" name="checkbox-sidebar" type="checkbox">
+  <label for="checkbox-sidebar" class="sidebar-custom-label" id="sidebar-label">
+    <i class="fal fa-bars sidebar-open"></i>
+    <i class="fal fa-times sidebar-close"></i>
+  </label>
+  <div class="sidebar-menu-container">
+    {% if CTX_FEATURES["QUARTIC_VIEW"] in features and features[CTX_FEATURES["QUARTIC_VIEW"]] %}
+    <span class="sidebar-menu-heading">QuarticView</span>
+    <ul class="sidebar-menu-list">
+      {% if CTX_FEATURES["DASHBOARD"] in features and features[CTX_FEATURES["DASHBOARD"]] %}
+      <li class="sidebar-menu-list-item">
+        <a class="sidebar-menu-list-link" href="{{site_url+'/contexalyze/my-dashboard'}}">
+          <i class="fal fa-window-restore sidebar-menu-list-icon"></i>
+          My Dashboard
+        </a>
+      </li>
+      {% endif %}
+      {% if CTX_FEATURES["RELIABILITY"] in features and features[CTX_FEATURES["RELIABILITY"]] %}
+      <li class="sidebar-menu-list-item">
+        <a class="sidebar-menu-list-link" href="{{site_url+'/quartic-view/reliability'}}">
+          <i class="fal fa-atom-alt sidebar-menu-list-icon"></i>
+          Reliability
+        </a>
+      </li>
+      {% endif %}
+      {% if CTX_FEATURES["ENERGY"] in features and features[CTX_FEATURES["ENERGY"]] %}
+      <li class="sidebar-menu-list-item">
+        <a class="sidebar-menu-list-link" href="{{site_url+'/quartic-view/energy'}}">
+          <i class="fal fa-bolt sidebar-menu-list-icon"></i>
+          Energy
+        </a>
+      </li>
+      {% endif %}
+      {% if CTX_FEATURES["THROUGHPUT"] in features and features[CTX_FEATURES["THROUGHPUT"]] %}
+      <li class="sidebar-menu-list-item">
+        <a class="sidebar-menu-list-link" href="{{site_url+'/quartic-view/throughput'}}">
+          <i class="fal fa-conveyor-belt-alt sidebar-menu-list-icon"></i>
+          Throughput
+        </a>
```

Comment:
Rather than hardcoding the menu can iterate over a data structure and get this information?

Reply:
Added new data structure in config file that will be used to iteration. I have also updated the template in the recent changes.